index.js

```diff
@@ -84,7 +75,7 @@ SmartBanner.prototype = {
 	constructor: SmartBanner,
 
 	create: function() {
-		var link = this.getStoreLink();
+		var link = this.options.url[this.type]
```

Comment:
@jbbuckner could getStoreLink check if something was passed to this.options and return that. If not, it could return what it originally did